tools/i18n-plugin/pom.xml

```diff
@@ -19,8 +19,8 @@
 
   <properties>
     <maven.core.version>3.6.0</maven.core.version>
```

Comment:
@wborn do you if this line has an effect at all? I could not find the define used in our code, nor as a documented parameter for a plugin....

Reply:
Yes it seems to be unused so you can probably safely remove it.

Reply:
done